packages/portal-proto/src/features/proteinpaint/GeneExpressionWrapper.tsx

```diff
@@ -57,7 +57,7 @@ export const GeneExpressionWrapper: FC<PpProps> = (props: PpProps) => {
     (arg: SelectSamplesCallBackArg) => {
       const cases = arg.samples.map((d) => d["cases.case_id"]);
       if (cases.length > 1) {
-        createSet({ values: cases });
+        createSet({ values: cases, intent: "user", set_type: "frozen" });
```

Comment:
no idea what to do with SJ's...

Reply:
🤷‍♀️

Reply:
I agree with what it is now

Reply:
I think the SJ ones should be portal if we are following the `on grounds that user just wants a cohort of stuff. it's us who made the stuff into a set.` logic.

Reply:
I'm not really in love with the fact that this logic is copy/pasted all over the St Jude apps though. It should probably be extracted to a new hook or component.

Reply:
> I think the SJ ones should be portal if we are following the on grounds that user just wants a cohort of stuff. it's us who made the stuff into a set. logic.

what do you think, @craigrbarnes ? if we wanna be consistent, i think it would be `portal`. (we can adjust during QA cycle too)

Reply:
Sure that is fine